tests/forms/test_validators.py

```diff
@@ -0,0 +1,54 @@
+
+import pytest
+import mock
+
+from wtforms.validators import ValidationError
+
+import dmutils.forms.validators
+
+
+@pytest.fixture
```

Comment:
Should these live in `conftest.py`? I'm not sure. We seem to create some fixtures in test files and some in `conftest.py`

Reply:
I think we put them in `conftest.py` when they would appear to be more generally useful, but keep them in-test-module when they're rather specific.

Reply:
My understanding from the pytest documentation is that fixtures that are shared across modules go in `conftest.py`.

I guess there would be a benefit to having a global email address fixture, currently this specific fixture is used only for my tests, so I don't think it needs to be in `conftest.py` until that changes.